server/services/voip/service.ts

```diff
@@ -209,10 +210,11 @@ export class VoipService extends ServiceClass implements IVoipService {
 			msg: comment,
 			groupable: false,
 		};
-
-		// TODO: Check room is voip
-		// TODO: Check event is valid voip event
 		// TODO: Add events (messageTypes) to IMessage interface
```

Comment:
@KevLehman I have fixed the rest of the part. I am not sure if you needed this comment. Please let me know.

Reply:
Fixed the todos 👀